docs/releasehistory.rst

```diff
@@ -8,12 +8,19 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 * ``micro`` increments represent bugfix releases or improvements in documentation
 
 
-Current Development
--------------------
-
 New features
 """"""""""""
-
+- `PR #363 <https://github.com/openforcefield/openforcefield/pull/363>`_: Implements
+  :py:class:`GBSAHandler <openforcefield.typing.engines.smirnoff.parameters.GBSAHandler>`,
+  which supports the
+  `GBSA tag in the SMIRNOFF specification <https://open-forcefield-toolkit.readthedocs.io/en/latest/smirnoff.html#gbsa>`_.
+  Currently, only GBSAHandlers with ``gb_model="OBC2"`` support
+  setting non-default values for the ``surface_area_penalty`` term (default ``5.4*calories/mole/angstroms**2``).
+  No model currently supports setting ``solvent_radius`` to any value other than ``1.4*angstroms``.
+  Experimental SMIRNOFF-format implementations of ``HCT``, ``OBC1``, and ``OBC2`` are included for testing
```

Comment:
It should be easy to include the SA penalty for these too! What's the limiting issue?

Reply:
The current implementations of `OBC1` and `HCT` only support a hard-coded value: https://github.com/openmm/openmm/blob/master/wrappers/python/simtk/openmm/app/internal/customgbforces.py#L372-L375

Reply:
We can easily fix that once we pull that file in.